packages/api/src/models.ts

```diff
@@ -1,6 +1,13 @@
 import { getModelForClass } from '@typegoose/typegoose';
-import { Consumer, Doc, Update } from '@a-la-fois/models';
+import { Consumer, Doc, Token, Update } from '@a-la-fois/models';
 
-export const DocModel = getModelForClass(Doc);
-export const UpdateModel = getModelForClass(Update);
-export const ConsumerModel = getModelForClass(Consumer);
+const options = {
+    schemaOptions: {
+        timestamps: true,
+    },
+};
+
+export const DocModel = getModelForClass(Doc, options);
```

Comment:
I think it's better to move timestamps in the models package
```ts
@ModelOptions({
    schemaOptions: {
        timestamps: true,
    }
})
export class Consumer {
    id: string;
    
    createdAt: Date;
    
    updatedAt: Date;

    @prop({ required: true })
    publicKey: string;

    @prop({ required: true })
    name: string;
}
```

In this case it will work in all services